.github/main.workflow

```diff
@@ -0,0 +1,53 @@
+workflow "Update data sources" {
+  on = "schedule(0 09 * * *)"
+  resolves = ["Fetch latest data sources"]
+}
+
+action "Fetch latest data sources" {
+  uses = "actions/npm@master"
+  args = "run update-data-sources"
+  env = {
+    NODE_OPTIONS = "--max_old_space_size=4096"
```

Comment:
This is only necessary in the `i18n` package I think, because we load so many large markdown files into memory.

Reply:
🔥'd